pom.xml

```diff
@@ -230,7 +230,7 @@
 			<dependency>
 				<groupId>org.javassist</groupId>
 				<artifactId>javassist</artifactId>
-				<version>3.19.0-GA</version>
```

Comment:
this change was the last one I merged, can you please do a `git pull --rebase upstream master` and push again